### PR TITLE
Update cursive to 0.14

### PR DESCRIPTION
**Pull request by gyscos**

### CODE REVIEW

**Comment by chrisvest** [comment]
Thanks for taking this on!

Travis failed on Windows. Restarted the 2 months old green build of master to see if its a change in the build environment. (The GitHub Actions builds have never been green; they're missing the ncurses dependency)...

Restarted old-master build failed, so looks like something in the build environment changed.

File: src/utilities.rs [hunk 1]
```diff
@@ -40,17 +39,19 @@ pub fn get_content(ev: &mut EditView) -> Rc<String> {
...
             eprintln!("File not found: {:#?}", name);
-        },
+        }
         ErrorKind::PermissionDenied => {
             eprintln!("Permission denied: {:#?}", name);
-        },
+        }
         _ => {
-            eprintln!("{}: {:#?}", error.description(), name);
+            eprintln!("{}: {:#?}", error, name);
```
Comment (chrisvest) [inline src/utilities.rs:54]: Is the description inferred via the Display trait?

**Review by chrisvest — COMMENTED** [review]
I had one question about the changes. Can you take a look?

File: src/utilities.rs (hunk 1 above)
Comment (gyscos) [reply src/utilities.rs:54]: Yes, there's a warning regarding the use of `Error:: description`.
https://doc.rust-lang.org/std/error/trait.Error.html#method.description

**Review by chrisvest — APPROVED** [review]

**Comment by gyscos** [comment]
Forgot to mention I also ran rustfmt on the affected files, which increased a bit the diff...
If you have a preferred style, you can try and see if it can be achieved with some rustfmt settings. Keeping a rustfmt.toml in the root of the repo will make autoformatters follow it :)

**Comment by gyscos** [comment]
> The GitHub Actions builds have never been green; they're missing the ncurses dependency

One solution could be to use the `crossterm` backend instead; it seems to be a reliable pure-rust cross-platform one.

**Comment by chrisvest** [comment]
I can give crossterm another try, and also see if https://github.com/gyscos/cursive/issues/352 has been fixed.

**Comment by chrisvest** [comment]
First impressions is that performance is worse, at least on Linux, with noticeable flicker even in release builds. We'll see how the build fares in #17.

**Pull request merged**